src/transformers/modeling_flax_roberta.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from transformers import BertConfig, RobertaConfig
+from transformers.modeling_flax_bert import FlaxBertModel
+
+
+class FlaxRobertaModel(FlaxBertModel):
```

Comment:
@LysandreJik @sgugger -> should we do the same as we've done in PT here? Copy paste the code using Sylvain's safety check?